models/tridimensional/docking_validation/chimera_generate_pams.py

```diff
@@ -1,10 +1,8 @@
-from __future__ import print_function
-
-import sys
-import os
 import argparse
-import math
 import errno
+import math
+import os
+import sys
 
 import chimera
```

Comment:
idk imports from within a script shouldn't have any impact on stuff outside the script. It would be good to remove if unneeded (by testing pdb creation with/without it).